bin/now-deploy.js

```diff
@@ -90,6 +92,7 @@ const help = () => {
     -f, --force               Force a new deployment even if nothing has changed
     -t ${chalk.underline('TOKEN')}, --token=${chalk.underline('TOKEN')}   Login token
     -L, --login               Configure login
+    -l, --links               Copy symlinks as symlinks
```

Comment:
Is it actually _not resolving symlinks_ ? I do agree that it needs re-wording, but not sure it's that one.